pkg/cmd/internal/issues/issues.go

```diff
@@ -403,22 +426,37 @@ func (p *poster) post(origCtx context.Context, formatter IssueFormatter, req Pos
 }
 
 func (p *poster) teamcityURL(tab, fragment string) *url.URL {
+	if p.TeamCityOptions == nil {
+		return nil
+	}
+	opts := p.TeamCityOptions
 	options := url.Values{}
 	options.Add("buildTab", tab)
 
-	u, err := url.Parse(p.ServerURL)
+	u, err := url.Parse(opts.ServerURL)
 	if err != nil {
 		log.Fatal(err)
 	}
 	u.Scheme = "https"
-	u.Path = fmt.Sprintf("buildConfiguration/%s/%s", p.BuildTypeID, p.BuildID)
+	u.Path = fmt.Sprintf("buildConfiguration/%s/%s", opts.BuildTypeID, opts.BuildID)
 	u.RawQuery = options.Encode()
 	u.Fragment = fragment
 	return u
 }
 
-func (p *poster) teamcityBuildLogURL() *url.URL {
-	return p.teamcityURL("log", "")
+func (p *poster) buildURL() *url.URL {
+	if p.Options.TeamCityOptions != nil {
+		u := p.teamcityURL("log", "")
+		return u
+	} else if p.Options.EngFlowOptions != nil {
+		u, err := url.Parse(p.Options.EngFlowOptions.ServerURL)
+		if err != nil {
+			log.Fatal(err)
```

Comment:
Just making sure `os.Exit` is the best recourse here?

Reply:
We're already doing it: see the `teamcityURL()` function, line 438 on this branch.

If you want me to add a little more `if err != nil { return err }`, let me know, I can do the mild refactor. Just trying to stay consistent with everything that's already here.

Reply:
> Just trying to stay consistent with everything that's already here.

Sounds good. I think it's a low risk (exit) path since the url is under our control; we can address later if that assumption no longer holds.